README.md

```diff
@@ -16,7 +16,23 @@ bitbucketApi.user.get().then(({ body }) => {
 
 ### `options`
 It is not necessary to provide any options at all (`Bitbucket` can be constructed with no argument).
- - `useXhr` (`Boolean`): If `true`, requests will be made using XMLHttpRequest. This is only available in a web browser, and will fail otherwise. This can be very useful in Electron for automatically resolving proxies and custom SSL certificates.
- - `proxy` (`String`): Defines a proxy to make requests against, instead of `api.bitbucket.org:443`. This option is _ignored_ when `useXhr` is active.
+ - `requesterFn` (`(options) => Promise<any>`): If provided, requests will be made using the function you provide. This is allows you to use your preferred http client. The `options` provided are `{ headers, hostname, method, path, url, body? }`. `body` is only provided on `POST` methods. Example:
```

Comment:
This doesn't reflect `prebuiltSend` where we take a response from the API and peel out a URL that the API gave us, e.g. the "next page" URL in a paginated response. In those cases where the package isn't building its own URL from scratch, pieces like `hostname` and `path` are not available. I think writing a caveat here explaining that would be sufficient.